game-core/src/main/java/games/strategy/engine/framework/startup/launcher/LocalLauncher.java

```diff
@@ -118,4 +123,33 @@ public static LocalLauncher create(
             null);
     return new LocalLauncher(gameSelectorModel, new PlainRandomSource(), pl, parent, launchAction);
   }
+
+  public static LocalLauncher create(
+      final List<Entry<String, String>> roleMapping,
+      final Predicate<String> isDisabled,
+      final GameData gameData) {
+    final Predicate<Entry<String, String>> enabledPredicate =
+        entry -> !isDisabled.test(entry.getValue());
```

Comment:
The parameter coming in seems to be a logical negation - can we sanely change that to `isEnabled`?

Second, Is this lamba a place where we can use the `Predicates.not(..)`?

Reply:
We could, but I'm pretty sure this would make the code longer and doesn't add to readability IMO

Reply:
Okay, it is a bit odd though as is, because:
1. enabled = !disabled
2. the predicate is not super clear, it's indirect, not clear why we would have a predicate or what the predicate really represents.

Perhaps confusing item (2) can be helped with naming. Instead of `enabledPredicate`, some (poor) suggestions, perhaps something like: `componentIsEnabled`, `componentIsNotDisabled`